src/Kiosk.Api/Services/NewsService.cs

```diff
@@ -50,6 +50,6 @@ private NewsResponse MapTranslatedNews(News news, Language language)
     {
         var newsList = await _newsRepository.GetManyNews(source, cancellationToken);
 
-        return newsList.Select(news => MapTranslatedNews(news, language));
+        return newsList?.Select(news => MapTranslatedNews(news, language));
```

Comment:
I'm not sure if you need `?` operator. It probably wouldn't be needed if you wrapped it in try catch block